Etabs_Adapter/CRUD/UpdateObject.cs

```diff
@@ -85,6 +90,32 @@ private bool UpdateObjects(IEnumerable<Node> nodes)
             return sucess;
         }
 
+        private bool UpdateObjects(IEnumerable<PanelPlanar> bhPanels)
+        {
```

Comment:
Don't think there is anything really wrong with what this method is implemented to do, but if added in like this, we probably want to make sure it is fully updating the object, so it can be used for updating things like Properties, Diaphragms and potentially even location in space.

Reply:
ya but thats something that I leave for the next project - one little chunk at a time well get to this full update functionality slow and steady